packages/frontend/src/components/Home/LandingPanel/index.tsx

```diff
@@ -34,19 +33,19 @@ const LandingPanel: FC<Props> = ({ hasSavedChart, userName, projectUuid }) => {
                     </Intro>
                 </WelcomeText>
 
-                <LinkButton
+                <StyledLinkButton
                     large
                     href={`/projects/${projectUuid}/tables`}
                     intent="primary"
-                    icon="database"
+                    icon="series-search"
                 >
                     Run a query
-                </LinkButton>
+                </StyledLinkButton>
             </LandingHeaderWrapper>
 
             <SpaceBrowser projectUuid={projectUuid} />
 
-            {hasSavedChart && <LatestDashboards projectUuid={projectUuid} />}
+            <LatestDashboards projectUuid={projectUuid} />
```

Comment:
cc'ing @PriPatel since they asked previously to not show the dashboard panel when there were no charts. You ok with this change ?

Reply:
Cool - yeah, I don think there should be any change to how it previously worked here for now.